src/index.ts

```diff
@@ -1,11 +1,11 @@
-import { authkitCallbackRoute } from './authkit-callback-route.js';
+import { handleAuth } from './authkit-callback-route.js';
 import { authkitMiddleware } from './middleware.js';
 import { getUser } from './session.js';
 import { getSignInUrl, signOut } from './auth.js';
 import { Impersonation } from './impersonation.js';
 
 export {
-  authkitCallbackRoute,
+  handleAuth,
```

Comment:
This kinda breaks backwards compatibility right? Wonder if that means we should release a major version.

Reply:
Typically I think that's fine given we're in the 0.x range.

From SemVer:

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.